test/duration/create.test.js

```diff
@@ -90,3 +90,41 @@ test("Duration.fromObject is valid if providing options only", () => {
   expect(dur.milliseconds).toBe(0);
   expect(dur.isValid).toBe(true);
 });
+
+//------
+// .fromDurationLike()
+//-------
+
+it("Duration.fromDurationLike returns a Duration from millis", () => {
+  expect.assertions(2);
+
+  const result = Duration.fromDurationLike(1000);
+
+  expect(result).toBeInstanceOf(Duration);
+  expect(result).toMatchInlineSnapshot(`"PT1S"`);
+});
+
+it("Duration.fromDurationLike returns a Duration from object", () => {
+  expect.assertions(2);
```

Comment:
You don't need to expect the assertion count. Asserting the count is important if you're doing async where the assertions are in callbacks that might not get called. But here we're not worried that our assertions won't happen

Reply:
Yep, fair enough, I just have a lint config in another project that complains if there's no `expect.assertions()` in every test. So it became a bit of habit.
Will remove :)